### DISCUSSION TITLE
Problems with visualization

### DISCUSSION THREAD

**Discussion by federicasibilla** [quantum-visualizations/qmsolve#12] [Q&A]
Hey! This repo is fantastic! Very useful while studying quantum...
Has anyone else had problems with the visualization part of the simulations for the time-dependent? I see the picture but time is stuck at zero and nothing happens :(

**Comment by rafael-fuente** [comment]
Some people have reported a problem with matplotlib animations when running scripts from a jupyter notebook. Are you running your script from a jupyter notebook? 
In that case, the problem may be the installation of matplolib.
